resources/default.rb

```diff
@@ -25,7 +25,7 @@
 def initialize(*args)
   super
   @action = :install
-  @run_context.include_recipe 'apt', 'java'
+  @run_context.include_recipe 'apt', 'java', 'pleaserun'
```

Comment:
Based on issue https://github.com/greglu/dropwizard-cookbook/issues/2 about the java cookbook, we probably don't need this line at all. In fact, we shouldn't even include the `apt` or `java` cookbooks as dependencies.